src/Fixer/ConstantNotation/NativeConstantInvocationFixer.php

```diff
@@ -0,0 +1,211 @@
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function isRisky()
+    {
+        return true;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function configure(array $configuration = null)
+    {
+        parent::configure($configuration);
+
+        $uniqueConfiguredExclude = \array_unique($this->configuration['exclude']);
+
+        // Case sensitive constants handling
+        $constantsToEscape = array_values($this->configuration['include']);
+        if (true === $this->configuration['fix_built_in']) {
+            $constantsToEscape = array_merge($constantsToEscape, \array_keys(\get_defined_constants()));
+        }
+        $constantsToEscape = \array_diff(
+            \array_unique($constantsToEscape),
+            $uniqueConfiguredExclude
+        );
+
+        // Case insensitive constants handling
+        static $caseInsensitiveConstants = ['null', 'false', 'true'];
+        $caseInsensitiveConstantsToEscape = [];
+        foreach ($constantsToEscape as $constantIndex => $constant) {
+            $loweredConstant = strtolower($constant);
+            if (in_array($loweredConstant, $caseInsensitiveConstants, true)) {
+                $caseInsensitiveConstantsToEscape[] = $loweredConstant;
+                unset($constantsToEscape[$constantIndex]);
+            }
+        }
+        $caseInsensitiveConstantsToEscape = \array_diff(
+            \array_unique($caseInsensitiveConstantsToEscape),
+            \array_map('strtolower', $uniqueConfiguredExclude)
+        );
+
+        // Store the cache
+        $this->constantsToEscape = \array_fill_keys($constantsToEscape, true);
+        \ksort($this->constantsToEscape);
+
+        $this->caseInsensitiveConstantsToEscape = \array_fill_keys($caseInsensitiveConstantsToEscape, true);
+        \ksort($this->caseInsensitiveConstantsToEscape);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function applyFix(\SplFileInfo $file, Tokens $tokens)
+    {
+        $indexes = [];
+        foreach ($tokens as $index => $token) {
+            $tokenContent = $token->getContent();
+
+            // test if we are at a constant call
+            if (!$token->isGivenKind(T_STRING)) {
+                continue;
+            }
+
+            $next = $tokens->getNextMeaningfulToken($index);
+            if ($tokens[$next]->equals('(')) {
+                continue;
+            }
+
+            $constantNamePrefix = $tokens->getPrevMeaningfulToken($index);
+            if ($tokens[$constantNamePrefix]->isGivenKind([CT::T_USE_TRAIT, T_CLASS, T_CONST, T_DOUBLE_COLON, T_EXTENDS, T_FUNCTION, T_IMPLEMENTS, T_INTERFACE, T_NAMESPACE, T_NEW, T_NS_SEPARATOR, T_OBJECT_OPERATOR, T_TRAIT, T_USE])) {
+                continue;
+            }
+
+            if (!isset($this->constantsToEscape[$tokenContent]) && !isset($this->caseInsensitiveConstantsToEscape[strtolower($tokenContent)])) {
+                continue;
+            }
```

Comment:
This fixer should also identify the constants imported with `use const` to avoid prefixing them. It would make the fixer much less risky (and it is necessary to support use statements if you want to later make this fixer configurable to allow injecting such use statements instead of injecting a `\`).

Reply:
Done.